projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -64,4 +76,6 @@ export default function RecordMeterBar( { totalCount, items = [] } ) {
 			</div>
 		</div>
 	) : null;
-}
+};
+
+export default RecordMeterBar;
```

Comment:
Since this component is moving to become a sharable component, this logic about showing it or not (based on total items) should not be made here.

This is something to be made by the component consumer, letting the component presentational without caring about outside logic since this kind of logic could differ easily.

Reply:
Great suggestion. Updated. Thank you

Reply:
@manzoorwanijk You probably need to update `projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/bar-chart.jsx` to add the removed logic there, not?

Reply:
That's not needed. That conditional rendering is already taken care of in `projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/index.jsx`